### DISCUSSION TITLE
[@types/dom-chromium-ai] The definition of the AISummarizerCapabilities does not match the object in Chrome Canary

### DISCUSSION THREAD

**Discussion by railsstudent** [DefinitelyTyped/DefinitelyTyped#71237] [Issues with a @types package]
```
interface AISummarizerCapabilities {
    supportsInputLanguage(languageTag: Intl.UnicodeBCP47LocaleIdentifier): AICapabilityAvailability;
}
```

The interface has `supportsInputLanguage` method but Chrome Canary 133 does not show it. Moreover,  the Chrome Canary has languageAvailable method in the same interface.

<img width="457" alt="Screenshot 2024-11-24 at 2 11 35 PM" src="https://github.com/user-attachments/assets/da4137c9-9185-4a11-932c-1b8669aa8421">

[a comment by an automated account is not shown]

**Comment by christianliebel** [comment]
Hi @railsstudent, thanks for your report. We have already prepared [a PR introducing this change](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70839/files#diff-ed45afcd489f9100e28aa666e02e81b24c748f5d9942639e422a2034e07170beR81). I ask for your patience until it is merged.

**Reply by railsstudent** [reply]
Awesome. I look forward to the changes and update my demos and slides in future talks.

**Reply by christianliebel** [reply]
The PR is now merged and a new package version is now available that contains the renamed method: [`@types/dom-chromium-ai@0.0.4`](https://www.npmjs.com/package/@types/dom-chromium-ai/v/0.0.4)

**Reply by railsstudent** [reply]
awesome.  I cannot wait to refactor my summarization and writer demos and blog about them.
